### DISCUSSION TITLE
GK2A data available on AWS

### DISCUSSION THREAD

**Discussion by bryanguarente** [blaylockbk/goes2go#90] [Ideas]
Hey Brian,
I see that GK2A data is now available on AWS and I think this would be another good feed to add to your library.  The setup is slightly different though as it isn't already converted to albedo or brightness temperature, so some work needs to be done to process the data.  I have code that works to convert it now, and I get normal looking imagery, but I didn't write up the access code for this as I wasn't as clear on how to do that with this library compared to Herbie.

Something to consider.
Bryan

**Comment by blaylockbk** [comment]
#91
